### ISSUE TITLE
WCAG 2.1.1: Elements that have scrollable content should be accessible by keyboard (.webchat__carousel-filmstrip.webchat__carousel-filmstrip--top-callout.react-film__filmstrip)

### ISSUE THREAD

**Issue by corinagum** [microsoft/BotFramework-WebChat#3814]
#### Issue
`Elements that have scrollable content should be accessible by keyboard` ([scrollable-region-focusable](https://accessibilityinsights.io/info-examples/web/scrollable-region-focusable))
#### Target application
[React App](https://corinagum.github.io/WebChat-release-testing/01.create-react-app/index.html)
#### Element path
.webchat__basic-transcript__activity--from-bot.webchat__basic-transcript__activity.webchat__basic-transcript__activity--acknowledged:nth-child(78) > .webchat__basic-transcript__activity-box > .webchat__carousel-layout.webchat--css-gdief-12j4dva.webchat--css-gdief-1xupiz0 > .react-film__main > .webchat__carousel-filmstrip.webchat__carousel-filmstrip--top-callout.react-film__filmstrip`

### Description: 
Carousels are navigable but contents are not, and are not described

### Repro: 
- layout double
- layout carousel
- slow
